.github/workflows/bot-ci-failure.yml

```
@@ -0,0 +1,87 @@
+  workflow_run:
+    workflows: ["Django Rest Framework Gis CI Build"]
+    types:
+      - completed
+
+permissions:
+  pull-requests: read
+  actions: read
+  contents: read
+
+concurrency:
+  group: group: ci-failure-${{ github.repository }}-${{ github.event.workflow_run.pull_requests[0].number || format('{0}:{1}', github.event.workflow_run.head_repository.full_name, github.event.workflow_run.head_branch) }} 
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)